src/tools/dev/scripts/bv_support/bv_icet.sh

```diff
@@ -108,17 +108,17 @@ function build_icet
     #
     CMAKE_BIN="${CMAKE_COMMAND}"
 
-    prepare_build_dir $ICET_BUILD_DIR $ICET_FILE
+    uncompress_src_file $ICET_SRC_DIR $ICET_FILE
```

Comment:
I made changes to this module recently. You should get the latest version.

Reply:
I have merged with develop.